src/stories/Library/link-with-icon/LinkWithIcon.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+import { LinkWithIcon } from "./LinkWithIcon";
+
+export default {
+  title: "Library / Link with icon",
+  component: LinkWithIcon,
+  decorators: [withDesign],
+  argTypes: {
+    href: {
+      defaultValue: "/",
+    },
+    linkText: {
+      defaultValue: "Følg prisoverrækkelsen live fra kl. 15:30",
+    },
+    linkType: {
+      control: {
+        type: "select",
+        options: ["internal", "external", "download", "search"],
+      },
+      defaultValue: "internal",
+    },
+  },
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/Zx9GrkFA3l4ISvyZD2q0Qi/Designsystem?type=design&node-id=7477-39110&mode=design&t=d1dz4FVEPRORqxUN-4",
+    },
+    layout: "full",
+  },
+} as ComponentMeta<typeof LinkWithIcon>;
+
+const Template: ComponentStory<typeof LinkWithIcon> = (args) => (
+  <LinkWithIcon {...args} />
+);
+
+export const Default = Template.bind({});
+
+const LinkWithIconListComponentTemplate: ComponentStory<
+  typeof LinkWithIcon
+> = () => (
+  <div>
+    <LinkWithIcon href="#" linkText="Internal Link" linkType="internal" />
+    <LinkWithIcon
+      href="https://example.com"
+      linkText="External Link"
+      linkType="external"
+    />
+    <LinkWithIcon
+      href="/download/some-file.pdf"
+      linkText="Download Link"
+      linkType="download"
+    />
+    <LinkWithIcon
+      href="/search?q=storybook"
+      linkText="Search Link"
+      linkType="search"
+    />
+  </div>
+);
+
+export const LinkList = LinkWithIconListComponentTemplate.bind({});
+LinkList.decorators = [(Story) => <div>{Story()}</div>];
```

Comment:
Please reconsider this decorator.

As far as I can tell it adds a wrapper `<div>` which seems superfluous when you already have one added on L42.

![ Link with icon - Link List ⋅ Storybook 2023-12-03 14-37-20](https://github.com/danskernesdigitalebibliotek/dpl-design-system/assets/73966/93faa0c2-7841-4dc7-ac10-65a3e9e2cb58)

Reply:
Good catch, oversight from me